samples/graphics/ultrahdr/src/main/java/com/example/platform/graphics/ultrahdr/display/CompressingUltraHDRImages.kt

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.platform.graphics.ultrahdr.display
+
+import android.graphics.Bitmap
+import android.graphics.BitmapFactory
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.annotation.RequiresApi
+import androidx.fragment.app.Fragment
+import com.example.platform.graphics.ultrahdr.databinding.CompressUltrahdrBinding
+import com.google.android.catalog.framework.annotations.Sample
+import java.io.ByteArrayOutputStream
+
+
+@RequiresApi(34)
+@Sample(
+    name = "Compressing UltraHDR Images",
+    description = "This sample demonstrates displaying an UltraHDR image in a Compose View and an Android View",
+    documentation = "https://developer.android.com/guide/topics/media/hdr-image-format",
+    tags = ["UltraHDR"],
+)
+
+class CompressingUltraHDRImages : Fragment() {
+
```

Comment:
Why is there all these random spaces? Can we clean this up just a little to not have them?

Reply:
Ack. Removed.